graphql/api.js

```diff
@@ -110,3 +110,54 @@ export const createAccount = async (email, name, company, role, date, type, note
 
   return data
 };
+
+//TODO: make query more dynamic for pagination
```

Comment:
do we still need to make the query dynamic?

Reply:
I think we definitely should in the future, but it's not necessary in the short term